using-valtimo/process/sendingmessages.md

```diff
@@ -0,0 +1,51 @@
+# Sending messages in BPMN
+
+Valtimo provides a way to send messages to another process/sub-process allowing not only
+correlation but association between processes and a Valtimo case.<br/>
```

Comment:
Why are we using `<br/>`? There's no need most of the time.

Also not sure what this sentence is meant to convey. Maybe the structure as a whole to:

When modeling a process, correlating a message (e.g. to start another process) does not do anything by default. Valtimo offers several methods to facilitate message correlation, which is intended to cover several different events and possible variations depending on the goal. These can be separated into starting a process (Message Start Events) and message catch events (Message Boundary Events and Message Intermediate Catch Events).

<<list of different use cases and the methods that they correspond to here>>

### How to use

<<explanation on how to use these methods in a BPMN>>